src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointClusterer.java

```diff
@@ -1,61 +1,70 @@
 package org.broadinstitute.hellbender.tools.spark.sv;
 
+import org.apache.spark.api.java.function.FlatMapFunction;
+import org.apache.spark.api.java.function.Function;
+
 import java.util.*;
-import java.util.function.Function;
 
 /**
  * A class that acts as a filter for breakpoint evidence.
  * It passes only that evidence that is part of a putative cluster.
  */
-public final class BreakpointClusterer implements Function<BreakpointEvidence, Iterator<BreakpointEvidence>> {
+public final class BreakpointClusterer implements Iterator<BreakpointEvidence> {
     private final int minEvidenceCount;
     private final int staleEventDistance;
```

Comment:
OK, as long as you thought about it. Definitely makes things much simpler without it.